model/lesson.js

```diff
@@ -122,6 +139,40 @@ async function getLessonById(data) {
     );
 }
 
+/*Fetches verses in a lesson based*/
+//ASSUMPTIONS: Verses in a Lesson are contigous
+async function getLessonVerses(data) {
+    console.log(data);
+    var invalid = utils.simpleValidation(data, {
+        lesson_id: "integer",
+    });
+    if (invalid) {
+        return invalid;
+    }
+
+    let lesson = await getLessonById(data);
+    if (!lesson.success) {
+        return lesson;
+    }
+
+    let lessonInfo = [];
+    let currentVerse = lesson.data[0].lesson_start_verse;
```

Comment:
Edge case --> what happens if there is no start or end verse?

Reply:
the values are assigned null, the resulting payload is an empty object. no special message is sent. I can add a check

Reply:
Hmm that's weird, the payload shouldn't be empty because we're querying by ID, right?